RadixWallet/Clients/TransactionClient/Models/TransactionModels.swift

```diff
@@ -132,14 +121,14 @@ public struct BuildTransactionIntentRequest: Sendable {
 	public let networkID: NetworkID
 	public let nonce: Nonce
 	public let manifest: TransactionManifest
-	public let message: Message
```

Comment:
`Message` does not contain a case called `None` anymore, any since we do not support message encryption yet anyway, I've simplified `Message` => `case plaintext(String)` instead of the more complex type, and made this optional.